docs/swarm/join-nodes.md

```diff
@@ -100,29 +105,29 @@ $ docker swarm join \
 This node joined a swarm as a manager.
 ```
 
-<!--TODO WIP
-Manager nodes use the listen address for cluster management communications. The
-other nodes on the swarm must be able to access the manager node on the
-IP address and port you specify for the listen address.
-
-Especially when there are multiple active network interfaces, you should
-you explicitly define the listen address when you add a manager node to the a
-swarm:
-
-  ```bash
-  docker swarm join \
-  --token <MANAGER-TOKEN> \
-  --listen-addr <NODE-IP>:<PORT> \
-  <MANAGER-IP>:<PORT>
-  ```
-this will change for https://github.com/docker/docker/pull/24237 ->>
-Replace <NODE-IP> with the IP address of the node that is joining the swarm.
-Replace <MANAGER-IP> with the address of the swarm manager.
-
-Only manager nodes use the listen address. If you specify `--listen-addr` for a
-worker node, the node only uses the listen address if it is promoted to a
-manager.
--->
+### Advertise address
+
+Manager nodes use an **advertise address** to allow other nodes in the swarm
+access to the API and overlay networking. If the system has multiple IP
+addresses, you must specify the correct  `--advertise-addr` to enable
+inter-manager communication and overlay networking:
+
+```bash
+$ docker swarm join \
+  --token <JOIN-TOKEN> \
+  --advertise-addr <NODE-IP>\
+  <MANAGER-IP>:2377
+```
+
+Replace <NODE-IP> with the listen address ip for the current node joining the
+swarm. Replace <MANAGER-IP> with the advertise address of the swarm manager
+node.
+
+Worker nodes only use the advertise address if you later promote them to the
+manager role.
+
+Refer to the `docker swarm init` [CLI reference](../reference/commandline/swarm_init.md)
+for more detail on the advertise address.
```

Comment:
CLI reference? should i spell out command line? Also, will remove this if it doesn't apply.